### DISCUSSION TITLE
Issue with docks #2?

### DISCUSSION THREAD

**Discussion by catalin-duta** [Keriew/augustus#655] [Q&A]
Not sure if related to #653 :
On the Valentia campaign, I am noticing a different issue with trading ships: they usually start at the easternmost dock, get full, then go around queueing at the other docks, docking for a second, then doing it again. I think they go through every dock before returning home.
Save attached; unstable build from ~5 commits back
[8Valentia1f.zip](https://github.com/Keriew/augustus/files/9426554/8Valentia1f.zip)

**Comment by crudelios** [comment]
Hi! This seems to be yet another bug...

To the list it goes!
